chapter02/n10.py

```diff
@@ -0,0 +1,15 @@
+"""
+行数をカウントせよ．確認にはwcコマンドを用いよ
+"""
+from sys import argv
+
+
+def count_len():
+    with open(argv[1]) as f:
+        counts = sum(1 for line in f)
```

Comment:
これくらいシンプルな処理なら代入しなくてもおけ
```suggestion
        return sum(1 for line in f)
```